app/models/state_file_nj_intake.rb

```diff
@@ -86,12 +91,18 @@ class StateFileNjIntake < StateFileBaseIntake
 
   encrypts :account_number, :routing_number, :raw_direct_file_data, :raw_direct_file_intake_data
 
-  enum household_rent_own: { unfilled: 0, rent: 1, own: 2, neither: 3 }, _prefix: :household_rent_own
+  enum household_rent_own: { unfilled: 0, rent: 1, own: 2, neither: 3, both: 4 }, _prefix: :household_rent_own
+
+  enum homeowner_home_subject_to_property_taxes: { unfilled: 0, yes: 1, no: 2}, _prefix: :homeowner_home_subject_to_property_taxes
+  enum homeowner_main_home_multi_unit: { unfilled: 0, yes: 1, no: 2}, _prefix: :homeowner_main_home_multi_unit
+  enum homeowner_main_home_multi_unit_max_four_one_commercial: { unfilled: 0, yes: 1, no: 2}, _prefix: :homeowner_main_home_multi_unit_max_four_one_commercial
+  enum homeowner_more_than_one_main_home_in_nj: { unfilled: 0, yes: 1, no: 2}, _prefix: :homeowner_more_than_one_main_home_in_nj
+  enum homeowner_shared_ownership_not_spouse: { unfilled: 0, yes: 1, no: 2}, _prefix: :homeowner_shared_ownership_not_spouse
 
   def disqualifying_df_data_reason
     w2_states = direct_file_data.parsed_xml.css('W2StateLocalTaxGrp W2StateTaxGrp StateAbbreviationCd')
     return :has_out_of_state_w2 if w2_states.any? do |state|
-      (state.text || '').upcase != state_code.upcase
+      !(state.text || '').casecmp(state_code).zero?
```

Comment:
[NIT] What's the purpose of changing this? I find the old one more readable.

Reply:
I'm guessing this was a rubocop autofix.

Reply:
So problem being, if I change it back, it'll just get changed again in the future

Reply:
Yep, if it's a rubocop fix, we can leave it.